bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -192,11 +207,16 @@ spec:
             --nb-address "tcp://${db_ip}:{{.OVN_NB_PORT}}" \
             --sb-address "tcp://${db_ip}:{{.OVN_SB_PORT}}" \
             --nodeport --gateway-mode local \
+            ${hybrid_overlay_flags} \
             --pidfile /var/run/openvswitch/ovnkube-node.pid \
-            --loglevel "${OVN_KUBE_LOG_LEVEL}" --logfile /dev/stdout
+            --loglevel 4  --logfile /dev/stdout
         env:
         - name: OVN_KUBE_LOG_LEVEL
           value: "4"
+        - name: OVN_EXTENSION_ENABLE
```

Comment:
@JacobTanenbaum let's change these to OVN_HYBRID_OVERLAY_ENABLE and OVN_HYBRID_OVERLAY_NET_CIDR

Reply:
done